.helm/build-images.sh

```
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+REPO_DIR=$(realpath "$SCRIPT_DIR"/..)
+
+# docker hub
+registry_uri=https://index.docker.io/v1/
+not_logged_in="echo \"You need to login to the docker registry $registry_uri\" && exit 1"
+jq -e ".auths | .\"${registry_name}\" | .auth" < ~/.docker/config.json || $not_logged_in
```

How is this line supposed to work? For me it echoes "You need to login...". However, I have properly authenticated with `docker login` and the build and push commands in the rest of this script run through perfectly fine.

Do we even need this line? `docker push` will anyway complain if you're not logged in, right?

My .env contains the following:
```bash
docker_hub_account=ursup
version=$(git rev-parse HEAD)
instance_name=${USER}
```

---

i wanted to fail early, but its an early optimization, so we can remove it